x/evm/keeper/msg_server.go

```diff
@@ -812,8 +813,10 @@ func (s msgServer) VoteConfirmDeposit(c context.Context, req *types.VoteConfirmD
 	if err != nil {
 		return nil, err
 	}
-	event.AppendAttributes(sdk.NewAttribute(types.AttributeKeyTransferID, transferID.String()))
-	s.Logger(ctx).Debug(fmt.Sprintf("confirmed deposit for %s with transfer ID %d", depositAddr.Address, transferID))
+
+	event = event.AppendAttributes(sdk.NewAttribute(types.AttributeKeyTransferID, transferID.String()))
+
+	s.Logger(ctx).Debug(fmt.Sprintf("confirmed deposit %s for %s with transfer ID %d (command ID %s)", pendingDeposit.TxID.Hex(), depositAddr.Address, transferID, types.TransferIDtoCommandID(transferID)))
```

Comment:
A bit weird to put command ID here since commands come later, but will be very helpful for debugging more quickly. Alternatively, we log the transfer ID along with the commandID later instead.